src/main/java/org/folio/innreach/batch/contribution/service/ContributionJobRunner.java

```diff
@@ -414,46 +426,55 @@ private void contributeInstance(UUID centralServerId, Instance instance, Statist
   }
 
   private void deContributeInstance(UUID centralServerId, Instance instance, Statistics stats) {
+    log.info("Initial: deContributeInstance centralServerId: {}, instanceId: {}, stats: {}", centralServerId, instance.getId(), stats);
     try {
       stats.addRecordsTotal(1);
       recordContributionService.deContributeInstance(centralServerId, instance);
       stats.addRecordsDeContributed(1);
       addRecordProcessed();
     }
     catch (ServiceSuspendedException | FeignException | InnReachConnectionException e) {
+      log.info("Initial: deContributeInstance exception occurred e: ", e);
       throw e;
     }
     catch (SocketTimeoutException socketTimeoutException) {
-      log.info("socketTimeoutException occur");
+      log.info("socketTimeoutException occur ", socketTimeoutException);
       throw new SocketTimeOutExceptionWrapper(socketTimeoutException.getMessage());
     }
     catch (Exception e) {
+      log.info("Initial: deContributeInstance exception occurred e: {} for instanceId: {}", e, instance.getId());
       instanceExceptionListener.logWriteError(e, instance.getId());
     } finally {
+      log.info("Initial: deContributeInstance finally added processed records and updated stats");
       stats.addRecordsProcessed(1);
       updateStats(stats);
     }
   }
 
   private void deContributeItem(UUID centralServerId, Item item, Statistics stats) {
+    log.info("Initial: deContributeItem centralServerId: {}, item id: {}, stats: {}", centralServerId, item.getId(), stats);
     try {
       stats.addRecordsTotal(1);
       recordContributionService.deContributeItem(centralServerId, item);
       stats.addRecordsDeContributed(1);
       addRecordProcessed();
     }
     catch (ServiceSuspendedException | FeignException | InnReachConnectionException e) {
+      log.info("Initial: deContributeItem exception occurred e: ", e);
       throw e;
     }
     catch (Exception e) {
+      log.info("Initial: deContributeItem exception occurred e: {} for item id: {}", e, item.getId());
       itemExceptionListener.logWriteError(e, item.getId());
     } finally {
+      log.info("Initial: deContributeItem finally added processed records and updated stats");
       stats.addRecordsProcessed(1);
       updateStats(stats);
     }
   }
 
   private void runOngoing(UUID centralServerId, BiConsumer<ContributionJobContext, Statistics> processor) {
+    log.info("Initial: runOngoing  centralServerId: {}, processor: {}", centralServerId, processor);
```

Comment:
Need to be careful here. Does processor have a toString implementation that is meaningful? If not it will simply output the data type which is kind of useless. We probably don't need to log this param unless I'm mistaken.

Reply:
Removed processor parameter because here we need centralServerId which is important in this case.